modules/ROOT/pages/_partials/sync/js-client-getting-started.adoc

```diff
@@ -1,10 +1,10 @@
 = Getting Started
 
-This describes how to start using JavaScript library for creating mobile and web applications using {sync-service} provided by Voyager Server. 
+This section describes how to use the Voyager JavaScript client SDK to create mobile and web applications that can communicate with the Voyager server application. 
 
 .Prerequisites
 
-* Your local `mobile-services.json` file is up-to-date.
+* The Voyager server application is running either locally or on an OpenShift cluster
```

Comment:
Instead of directly using `Voyager server` you can use `{sync-server}`. It would be nice to use that just in case we need to rename it later.

Reply:
yes, will fix it

Reply:
+1